### ISSUE TITLE
BUILD - repository changes: check only folders that will be commit to the repository

### ISSUE THREAD

**Issue by jorgetutor** [Metadrop/drupal-artifact-builder#8]
There is a protection to avoid artifact misalignments that checks git status to identify any critical difference.
It is needed that only the folders that will be used in the repository are analyzed.

Example:
`
```
 There are changes in the repository (changed and/or untracked files), please run this artifact generation script with folder tree clean. Files changed  `
 M .ddev/config.yaml                     
```

**Comment by omarlopesino** [comment]
On it.

**Comment by omarlopesino** [comment]
Released in 1.5.0-alpha1 a version that only checks artifact content folders. 

Testing it, if it works okay, it will be released at 1.5.0.

**Comment by omarlopesino** [comment]
Version 1.5.0-alpha2 works fine. Creating stable release.

**Comment by omarlopesino** [comment]
Released at 1.5.0. Everything works fine. I like this improvement, thanks for suggesting it!
